runtime/metrics/Cargo.toml

```diff
@@ -7,8 +7,9 @@ edition.workspace = true
 [dependencies]
 sp-std = { package = "sp-std", git = "https://github.com/paritytech/substrate", branch = "master", default-features = false}
 sp-tracing = { git = "https://github.com/paritytech/substrate", branch = "master", default-features = false }
-parity-scale-codec = { version = "3.3.0", default-features = false }
+parity-scale-codec = { version = "3.3.0", default-features = false, features = ["full"]}
```

Comment:
This feels wrong to me. I've added `features = ["full"]`  to overcome this error:
```
  error[E0277]: the trait bound `f64: WrapperTypeEncode` is not satisfied
    --> /home/ceco/parity/src/polkadot-runtime-metrics/primitives/src/v4/metrics.rs:23:10
     |
  23 | #[derive(Encode, Decode)]
     |          ^^^^^^ the trait `WrapperTypeEncode` is not implemented for `f64`
     |
     = help: the following other types implement trait `WrapperTypeEncode`:
               &T
               &mut T
               Box<T>
               Cow<'a, T>
               bytes::bytes::Bytes
               parity_scale_codec::Ref<'a, T, U>
               sp_application_crypto::Vec<T>
               sp_core::Bytes
     = note: required for `f64` to implement `parity_scale_codec::Encode`
     = note: this error originates in the derive macro `Encode` (in Nightly builds, run with -Z macro-backtrace for more info)

  error[E0277]: the trait bound `f64: WrapperTypeDecode` is not satisfied
    --> /home/ceco/parity/src/polkadot-runtime-metrics/primitives/src/v4/metrics.rs:31:19
     |
  31 |     ObserveHistogram(f64),
     |                      ^^^ the trait `WrapperTypeDecode` is not implemented for `f64`
     |
     = help: the following other types implement trait `WrapperTypeDecode`:
               Box<T>
               sp_core::Bytes
     = note: required for `f64` to implement `parity_scale_codec::Decode`
```
For some reason the code compile just fine with `runtime-metrics` feature but I can't understand what it uses/enables to avoid the error above.

Help with this will be much appreciated!

Reply:
Yes, that worked. Thanks @bkchr